src/index.js

```diff
@@ -60,7 +60,7 @@ class ProcessHtml {
         const ignoredFromPartial = ignoreLinksFromPartialMatches.filter(partial => href.indexOf(partial) >= 0);
 
         if (ignoreLinks.indexOf(href) < 0 && ignoredFromPartial.length === 0) {
-          source += `\nimport '${path}';\n`;
+          source += `\nimport '${path.replace(/\\/g, '\\\\')}';\n`;
```

Comment:
```
import { sep } from 'path'
...
const normalize = (str) => str.split(path.sep).join('/')
```
If I not mistaken webpack picks up unix paths with '/' accross all systems, but not :100:, otherwise `JSON.stringify(source)` ?